app/src/androidTest/java/com/github/fribourgsdp/radio/HostGameHandlerTest.kt

```diff
@@ -37,7 +38,7 @@ class HostGameHandlerTest {
         handler.handleSnapshot(mockSnapshot)
 
         // Wait for the task of the database to execute
-        Thread.sleep(1000)
+        Thread.sleep(sleepingTime)
```

Comment:
I didn't see it in the previous pull requests, but isn't there a way to do this more cleanly without having to put a Thread.sleep?

Reply:
For this you have to see with @VictorSchneuwly but it seemed to be obligatory as a way to let the application have the time to do the things it needs to do. Locally, it works with 1 millisecond as sleepingTime, but it failed once on Cirrus so I increased the value a litlle bit.